### PR TITLE
F/toast

### PR DESCRIPTION
**Pull request by asibilia**
Adds Toast element and documentation. Issue #18 

##### **What did you change?**
* Split into NovoToast and ToastService
* Added Toast element and documentation

##### **Reviewers**
* @jgodi 

##### **Checklist**
* [x] Changes are limited to a single goal (no scope creep)
* [x] Code can be automatically merged (no conflicts)
* [x] Code follows the standards laid out in the [contributing guide](https://github.com/bullhorn/novo-elements/blob/master/CONTRIBUTING.md)
* [x] Passes all existing automated tests
* [x] New functions include new tests
* [x] New functions are documented (with a description, list of inputs, and expected output)
* [x] Visually tested in supported browsers and devices

### CODE REVIEW

**Comment by jgodi** [comment]
The e2e test stub fails build. Also if we are not able to merge/squash you will need to set your git to rebase by default to get away from the `Merge branch 'master' of https://github.com/bullhorn/novo-elements`, it can affect the history.

File: src/elements/toast/Toast.js [hunk 1]
```diff
@@ -0,0 +1,174 @@
+import { Component, Injectable, ApplicationRef, DynamicComponentLoader, ElementRef, Injector } from 'angular2/core';
+import { CORE_DIRECTIVES } from 'angular2/common';
+
+@Injectable()
+export class NovoToaster {
```
Comment (jgodi) [inline src/elements/toast/Toast.js]: Can we call this `ToastService`? and maybe move it into its own file `src/elements/toast/ToastService.js` with it's own stubbed spec. The documentation can still live in the same README

File: src/elements/toast/Toast.js [hunk 2]
```diff
@@ -0,0 +1,56 @@
+import { Component, ElementRef } from 'angular2/core';
+import { CORE_DIRECTIVES } from 'angular2/common';
+
+@Component({
+    selector: 'novo-toast',
+    inputs: ['theme', 'icon', 'title', 'message', 'time', 'position'],
```
Comment (jgodi) [inline src/elements/toast/Toast.js]: Should `show` be an input? Or will the user user `*ngIf` to hide/show the toast?

Comment (asibilia) [inline src/elements/toast/Toast.js]: I think we should let the user `*ngIf` for now but we can add a feature request issue for adding `show` as an input that is tied to `time`. I took off `time` and `position` on static `novo-toast` elements for now.

File: src/novo-elements.js [hunk 3]
```diff
@@ -10,13 +11,13 @@ import { NOVO_SELECT_ELEMENTS } from './elements/select';
...
 export * from './elements/button';
 export * from './elements/tabs';
+export * from './elements/toast';
 export * from './elements/switch';
 export * from './elements/card';
 export * from './elements/loading';
 export * from './elements/select';
 export * from './elements/tooltip';
 export * from './elements/drawer';
-export * from './elements/switch';
```
Comment (jgodi) [inline src/novo-elements.js]: Bad merge?

Comment (asibilia) [inline src/novo-elements.js]: don't believe so, switch is in line 13, this one is a dup

**Pull request merged**